### PR TITLE
Add screenshots, tweak phrasing in the "Updating to Hosted" page

### PR DESCRIPTION
**Pull request by ramonh**

### CODE REVIEW

File: docs/updating.md [hunk 1]
```diff
@@ -76,22 +50,46 @@ module.exports = function override(config, env) {
...
+        /* other plugins may be listed here */
+        ["@codesee/instrument", {hosted: true}]
+      ],
+    }
+  }
+```
+
+## 3. Reset localStorage
+
+If you're updating from a version of CodeSee with local storage to hosted, you'll next need to open your application in your browser, making sure that the CodeSee floating action button is visible:
```
Comment (jleven) [inline docs/updating.md:84]: I don’t think this is an “if”. I think everyone needs to do this.

Comment (jleven) [reply docs/updating.md:84]: “You’ll next need to clear localStorage”

**Review by jleven — COMMENTED** [review]
Looks great!! Just one or two things I noticed.

**Comment by ramonh** [comment]
Thank you so much, @jleven! I've tweaked the wording as recommended.

**Review by jleven — APPROVED** [review]
Love it!

**Pull request merged**